client/deposits/list/index.tsx

```diff
@@ -210,9 +210,9 @@ export const DepositsList = (): JSX.Element => {
 	const downloadable = !! rows.length;
 
 	const onDownload = async () => {
-		recordEvent( 'wcpay_deposits_download', {
-			exported_deposits: rows.length,
-			total_deposits: depositsSummary.count,
+		recordEvent( 'wcpay_csv_export_click', {
+			location: 'payouts',
```

Comment:
Ideally we'd use a real path string or woo "page" for this prop. I bet there are similar props in Woo core and maybe even WooPayments – a prop that says what page/screen the event occurred on. @nagpai can you research in WooPayments and Woo core to find similar props? Prop might be named `location`, `screen`, `path` etc.

Reply:
Other events might also use `source` prop for this 🤔 

If we can use a woo path/page name directly, then we don't need to hard-code a string here (i.e. avoid defining a one-off schema).

Reply:
I have included a `source` prop that captures the path of the page where the button is clicked. Excellent idea to not have this hardcoded 🙌🏼